test/test_other.py

```diff
@@ -12881,3 +12881,14 @@ def test_dbg(self):
                  'ReferenceError: dbg is not defined',
                  emcc_args=['-DNDEBUG', '-sASSERTIONS=0'],
                  assert_returncode=NON_ZERO)
+
+  def test_googletest(self):
```

Comment:
Should we have logic to skip this if `googletest` isn't installed? Otherwise I think this would be the first git submodule we strictly require for core tests (the posix test suite must be run explicitly).

Reply:
Unlike the posixtest example, the fact that this test depends on a git submodule doesn't effect the entire suite or prevent other tests from running.

I feel like anyone who wants to run the entire test suite should have up-to-date git submodules.  

For folks such as emsdk users who might just need/want to run a single tests that still works.  Such users already can't run the entire test suite because we prune large amount of the test suite when we create the emsdk.

Reply:
On other words, I don't think there are any users who would benefit from having this test auto-skip, since we have two classes of folks who want to run tests:

1. emscripten developers with full checking
2. emsdk users with a partial copy of the test suite

(1) folks should already be grabbing the git submodules.
(2) folks are already severely limits in the tests they can run.

Reply:
Good points, sgtm.